### DISCUSSION TITLE
First Trade Kills?

### DISCUSSION THREAD

**Discussion by Dying-Ducks** [akiver/cs-demo-manager#821] [General]
In the overview of a demo one can find the columns "First Trade Kills (Kill revenged within 5s after a teammate died)" and "First Trade Deaths (Deaths revenged within 5s after a teammate died)".

At first it seems logical to include both as its legit to want to know if the opening duels are getting traded.

But if one think about "First Trade Kills" the concept is getting weird:
As the "First Kill" is the first kill in a round you can never revenge anybody by the kill since nobody died yet. Otherwise, the first kill wouldn’t be the first kill anymore. 

In other words: the column "First Trade Kills" is nonsensical in the current state, as the first kill is never a "trade kill" by the metric "isTradeKill".

So to make sense for the first kill, we would need a different parameter next to "isTradeKill".
"isTradeKill" answered the question "Did this kill traded something?".
And we would need to answer the question "Will this kill get traded?".
